Examples/Physics_applications/spacecraft_charging/inputs_test_rz_spacecraft_charging_picmi.py

```diff
@@ -82,49 +82,34 @@ def save_normalized_vacuum_Efields(
         self.spacecraft_capacitance = 1.0 / q_v  # the potential was set to 1V
 
         warpx = sim.extension.warpx
-        fields = warpx.multifab_register()
+        multifab_register = warpx.multifab_register()
 
-        phi = fields.get("phi_fp", 0)
-        Er = fields.get("Efield_fp", self.dir_r, 0)
-        Ez = fields.get("Efield_fp", self.dir_z, 0)
+        phi = multifab_register.get("phi_fp", 0)
+        Er = multifab_register.get("Efield_fp", self.dir_r, 0)
+        Ez = multifab_register.get("Efield_fp", self.dir_z, 0)
         # Allocate the fields `normalized_Er`, `normalized_Ez`, and `normalized_phi
         # in WarpX's multifab register. This allows to get these fields at later
-        # iterations with fields.get( ... ).
+        # iterations with multifab_register.get( ... ).
         # These new fields are automatically redistributed when doing load balancing.
-        normalized_Er = fields.alloc_init(
-            "normalized_Er",
-            0,
-            Er.box_array(),
-            warpx.DistributionMap(0),
-            1,
-            Er.n_grow_vect,
-            0.0,
-            True,
-            True,
+        normalized_Er = fields.MultiFabWrapper(
```

Comment:
@RevathiJambunathan I've added a way of adding new MultiFabs to the wrapper and updated this to use it. How does this look? Note that I changed the variable name used for the registry since using `fields` conflicts with the name of the `fields` module where the wrappers are.